src/fiskaltrust.Launcher/ProcessHost/ProcessHostMonarcStartup.cs

```diff
@@ -51,22 +54,21 @@ public ProcessHostMonarcStartup(ILoggerFactory loggerFactory, ILogger<ProcessHos
                 }
             });
         }
-
-
+        
         protected override async Task ExecuteAsync(CancellationToken cancellationToken)
         {
             _lifetime.ApplicationLifetime.ApplicationStopping.Register(() => _logger.LogInformation("Shutting down launcher."));
             cancellationToken.Register(() => _kestrelReady.TrySetCanceled());
 
             StartupLogging();
 
-            if (_launcherConfiguration.LauncherPort == 0)
+            if (string.IsNullOrEmpty(_launcherConfiguration.LauncherServiceUri))
```

Comment:
You can remove the whole code in this if. It's functionality was to check if kestrel is ready so we can overwrite the launcher port with the actual port if it was `0` (using dynamic binding). As we don't have that anymore just remove it.